dockers/docker-fpm-quagga/bgpd.conf.j2

```diff
@@ -65,6 +69,21 @@ router bgp {{ minigraph_bgp_asn }}
 {% endif %}
 {% endfor %}
 {% endblock bgp_sessions %}
+{% block bgp_peers_with_range %}
+{% if deploymentId and deploymentId_asn_map[deploymentId] %}
```

Comment:
maybe it is better to make the failure explicit? when there is bgp peers with range, then we need to generate the passive peer. if deploymentId is not available, it is probably good to make the failure explicitly?